### PR TITLE
Refactor hide-issue-events

### PR DESCRIPTION
**Pull request by wxiaoguang**

### CODE REVIEW

**Comment by wxiaoguang** [comment]
ps: I haven't chosen the the i18n strings carefully, they might need to be updated.

**Comment by qwerty287** [comment]
@wxiaoguang thanks! I'll review and merge it later.

File: models/user/setting.go [hunk 1]
```diff
@@ -31,8 +31,8 @@ func init() {
 	db.RegisterModel(new(Setting))
 }
 
-// GetSettings returns specific settings from user
-func GetSettings(uid int64, keys []string) (map[string]*Setting, error) {
+// GetUserSettings returns specific settings from user
```
Comment (qwerty287) [inline models/user/setting.go:34]: I understand these changes, however I don't know if it's good to do this in this PR and not providing a separate PR because this is not related to the feature request of the original PR.

Comment (wxiaoguang) [reply models/user/setting.go:34]: There was no other user (caller) of the user setting package. So I think it's fine to refactor together, to reduce unnecessary PRs and save time. But yes, it can be in a separate PR, I don't know how others think about this.

**Comment by qwerty287** [comment]
Thank you!
